core.js

```diff
@@ -5,10 +5,7 @@ var includes = require('lodash.includes');
 var assign = require('lodash.assign');
 var qs = require('qs');
 var mediaType = require('media-type');
-
-// TPromise stands for then/promise, used as a way to avoid naming collision with native Promise implementation
-// at this point, this is largely to avoid jshint from complaining
```

Comment:
Good question. JSHint just doesn't give me any warning anymore. Maybe it was updated?